admin/class-botamp-admin.php

```diff
@@ -262,6 +276,19 @@ public function register_setting() {
 			array( 'label_for' => $this->option( 'entity_url' ) )
 		);
 
+		if ( in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', get_option( 'active_plugins' ) ) ) ) {
+			add_settings_field(
+				$this->option( 'order_notifications' ),
+				__( 'Order notifications', 'botamp' ),
+				array( $this, 'order_notifications_cb' ),
+				$this->plugin_name,
+				$this->option( 'general' ),
+				array( 'label_for' => $this->option( 'order_notifications' ) )
+			);
+
+			register_setting( $this->plugin_name, $this->option( 'order_notifications' ) );
+		}
```

Comment:
is it possible to abstract all the woocommerce related code from BotampAdmin to a BotampWoocommerceAdmin?